.gitignore

```diff
@@ -19,6 +19,9 @@ target/
 *.iml
 *.ipr
 
+### Local files ###
+*.DS_Store
```

Comment:
In order for this to be ignored, first you need to commit the change to the gitignore file. As you can see the .DS_Store file is not ignored.

Reply:
not done but I will find the way how to fix it